src/common/urls.ts

```diff
@@ -0,0 +1,14 @@
+import { Linking } from 'react-native';
+
+export const enforceWebURLProtocol = (url: string) => {
+  const allowedProtocols = ['https://', 'http://', 'mailto:'];
```

Comment:
Any reason to include `http://`?  I think it'd be better to leave it out for now, until a customer requires it.

Reply:
Oh, this is just a move, not new code - disregard.